tests/__init__.py

```diff
@@ -10,16 +10,11 @@
 os.environ['XCCDF_VALUE_SOLUTION_FILE'] = solution_txt
 os.environ['XCCDF_VALUE_REPORT_DIR'] = dirname
 os.environ['XCCDF_VALUE_CURRENT_DIRECTORY'] = os.path.join(os.path.dirname(__file__), "..")
-os.environ['XCCDF_RESULT_ERROR'] = "3"
-os.environ['XCCDF_RESULT_FAILED'] = "2"
-os.environ['XCCDF_RESULT_FAIL'] = "2"
-os.environ['XCCDF_RESULT_FIXED'] = "1"
-os.environ['XCCDF_RESULT_NEEDS_INSPECTION'] = "1"
-os.environ['XCCDF_RESULT_NEEDS_ACTION'] = "1"
-os.environ['XCCDF_RESULT_NOT_APPLICABLE'] = "10"
+os.environ['XCCDF_RESULT_FAIL'] = "102"
+os.environ['XCCDF_RESULT_FIXED'] = "109"
+os.environ['XCCDF_RESULT_NOT_APPLICABLE'] = "105"
```

Comment:
Removed values as XCCDF_RESULT_ERROR, ...,  makes traceback during testing. Tests should work.